File: compat/corpus/dynamic.ts
export function dispatchEvent(event: string): string {
  return event.toUpperCase();
}

function invoke(callback: (event: string) => string): string {
  return callback("created");
}

const callbacks = { created: dispatchEvent };
const alias = dispatchEvent;
const callbackResult = invoke(dispatchEvent);
const directResult = dispatchEvent("updated");

export { alias, callbackResult, callbacks, directResult };

File: compat/corpus/dynamic.py
from collections.abc import Callable


def dispatch_order(order_id: str) -> str:
    return order_id.upper()


def invoke(callback: Callable[[str], str]) -> str:
    return callback("order-1")


CALLBACKS = {"order": dispatch_order}
alias = dispatch_order
callback_result = invoke(dispatch_order)
direct_result = dispatch_order("order-2")
